samcli/local/docker/lambda_debug_settings.py

```diff
@@ -81,6 +81,10 @@ def get_debug_settings(debug_port, debug_args_list, _container_env_vars, runtime
                 entry + ["/var/runtime/bootstrap"] + debug_args_list,
                 container_env_vars={"_AWS_LAMBDA_DOTNET_DEBUGGING": "1", **_container_env_vars},
             ),
+            Runtime.dotnet5.value: DebugSettings(
+                entry + ["/var/runtime/bootstrap"] + debug_args_list,
+                container_env_vars={"_AWS_LAMBDA_DOTNET_DEBUGGING": "1", **_container_env_vars},
+            ),
```

Comment:
I am not sure but can you confirm that we support debugging for image option as well (not just for zip)?

Reply:
I think yes, Dockerfile has [SAM_BUILD_MODE](https://github.com/aws/aws-sam-cli-app-templates/blob/master/dotnetcore3.1-image/cookiecutter-aws-sam-hello-dotnet-lambda-image/%7B%7Bcookiecutter.project_name%7D%7D/src/HelloWorld/Dockerfile#L16) which in my opinion used for debugging.

Reply:
I see but were you able to start a debugging session by running `sam local invoke -d {debugport}` and see the execution have been halted?

Reply:
This is only used for zips if I recall correctly. This shouldn't be needed at all.

Reply:
Without it, `make pr` fails 
```
E           samcli.local.docker.lambda_debug_settings.DebuggingNotSupported: Debugging is not currently supported for dotnet5.0

samcli/local/docker/lambda_debug_settings.py:137: DebuggingNotSupported
```